src/components/VmActions/index.js

```diff
@@ -106,12 +106,18 @@ const VmActions = ({ vm, isOnCard = false, onGetConsole, onShutdown, onRestart,
     confirmText: 'Suspend?',
     cancelText: 'Cancel',
   }
-
+  let consoleProtocol = ''
+  if (vm.get('consoles').size) {
+    consoleProtocol = 'Open '
+    consoleProtocol += (vm.get('consoles').find(c => c.get('protocol') === 'spice') ||
```

Comment:
please extract the inner expression concerned with looking up the correct protocol into a variable and then use ES6 string interpolation instead of manual `+` concatenation:

```javascript
protocol = vm.get(...) || ...
consoleProtocol = `Open ${protocol} Console` 
```